inflammation/models.py

```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
```

Comment:
Variable names unclear for mmm and devs. 
return variable is called s_dev2 - is the function returning standard deviation squared? Description says it returns the standard deviation

Reply:
Changed accordingly